.doc_gen/cross-content/cross_SNSSubPub_Java_block.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd"[
+    <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+    %phrases-shared;
+]>
+<block>
+    <para>
+        Shows how to use the &SNSlong; Java API to
+        create a web application that has 
+        subscription and publish functionality. 

```

Comment:
I'd include the translate functionality as well in this description.

Reply:
fixed